ast/src/main/java/com/graphicsfuzz/common/typing/TyperHelper.java

```diff
@@ -840,13 +841,81 @@ public static Type resolveTypeOfMul(Type lhsType, Type rhsType) {
 
     // 8.8: Integer Functions
 
+    if (shadingLanguageVersion.supportedIntegerFunctions()) {
+      getBuiltinsForGlslVersionInteger(builtinsForVersion);
+    }
+
     // 8.13: Fragment Processing Functions (only available in fragment shaders)
 
     // 8.14: Noise Functions - deprecated, so we do not consider them
 
     return builtinsForVersion;
   }
 
+  private static void getBuiltinsForGlslVersionInteger(
+      Map<String, List<FunctionPrototype>> builtinsForVersion) {
```

Comment:
Change TODO to TODO(GitHub issue id)
Remove TODO from second line, but indent enough so that IDEA still highlights the line in yellow.